src/CMakeLists.txt

```diff
@@ -759,6 +759,7 @@ if( MIOPEN_BACKEND STREQUAL "OpenCL")
     target_link_libraries( MIOpen PUBLIC ${OPENCL_LIBRARIES} )
     list(APPEND PACKAGE_DEPENDS PACKAGE OpenCL)
 elseif(MIOPEN_BACKEND STREQUAL "HIPOC" OR MIOPEN_BACKEND STREQUAL "HIP")
+    target_link_libraries(MIOpen PRIVATE hiprtc)
```

Comment:
Not a bit deal but can we follow the format of the other similar lines?

Reply:
Done